### PR TITLE
Spiked bats

### PR DESCRIPTION
**Pull request by asskek**
Adds spiked baseball bats.

### CODE REVIEW

File: hippiestation/code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -69,3 +69,7 @@
 /obj/item/storage/box/syndie_kit/firesuit/PopulateContents()
 	new /obj/item/clothing/suit/fire/atmos/syndicate(src)
 	new /obj/item/clothing/head/hardhat/atmos/syndicate(src)
+
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/storage/uplink_kits.dm]: ?

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -73,7 +73,31 @@
...
+	armour_penetration = 20
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/melee/baseball_bat/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stack/rods))
+		var/obj/item/melee/baseball_bat/spike/B = new /obj/item/melee/baseball_bat/spike
+
+		remove_item_from_storage(user)
+		qdel(I)
+		qdel(src)
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/weaponry.dm]: If you qdel src,the proc will stop running. You must do this at the end, as if it was a return

File: hippiestation/code/modules/crafting/recipies.dm [hunk 3]
```diff
@@ -41,4 +41,5 @@
 		         /obj/item/screwdriver)
 	time = 150
 	category = CAT_WEAPONRY
-	subcategory = CAT_WEAPON
\ No newline at end of file
+	subcategory = CAT_WEAPON
```
Comment (Carbonhell) [inline hippiestation/code/modules/crafting/recipies.dm:44]: ?

**Review by Carbonhell — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/crafting/recipies.dm (hunk 3 above)
Comment (asskek) [reply hippiestation/code/modules/crafting/recipies.dm:44]: Oh, this is a leftover from when I was trying to make it a tablecrafting item. Forgot to remove that.

File: hippiestation/code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (asskek) [reply hippiestation/code/game/objects/items/storage/uplink_kits.dm]: Not sure what's happening here, did I delete something on accident?

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 2 above)
Comment (asskek) [reply hippiestation/code/game/objects/items/weaponry.dm]: Alright, I'll do that when I get home from work. Thanks man.

Comment (Manly-Dwarf) [reply hippiestation/code/game/objects/items/weaponry.dm]: Pretty sure this is just copy+pasted Spear Code, you nimmy.

**Comment by Pyko1** [comment]
would be neat if you could use chem rods, to give the bat a chem inject trait.

File: hippiestation/code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (Carbonhell) [reply hippiestation/code/game/objects/items/storage/uplink_kits.dm]: no, you added a lot of new lines 4noraisin, delete them and make the file not show here

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 2 above)
Comment (Carbonhell) [reply hippiestation/code/game/objects/items/weaponry.dm]: just doublechecked,i confirm it's potentially buggy behaviour. fix

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
+1

**Comment by GuyonBroadway** [comment]
>GA's still think thier +1s count here

**Comment by Pyko1** [comment]
+1

**Comment by JohnGinnane** [comment]
@asskek pls

**Comment by Carbonhell** [comment]
@asskek wake up nerd

**Comment by bardofrage** [comment]
+1

**Comment by praisenarsie** [comment]
stale, ping for reopen

**Comment by Carbonhell** [comment]
Since asskek went MIA i will complete the pr later once i get home

**Comment by Carbonhell** [comment]
Will fix tomorrow cause web editor bugs out on mobile, i spotted another issue aswell(rods shouldn't be qdelled, they should use use())

**Comment by Carbonhell** [comment]
Pr cannot be merged due to a conflict we cannot resolve, someone please remake this

**Comment by JohnGinnane** [comment]
🆗

**Pull request closed without merging**